buildlib/az-GitHubDraft.yml

```diff
@@ -0,0 +1,47 @@
+# Create an empty draft to avoid race condition in distro release
```

Comment:
how is it related to ARM build?

Reply:
It's an existing job, just separated into 2 platforms.